nbs/index.ipynb

```diff
@@ -188,10 +181,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dlmo_f = f_model.dlmos(trajectory)\n",
-    "dlmo_kj = kj_model.dlmos(trajectory)\n",
```

Comment:
Note the bug here where `dlmo_kj` is calculated on Forger's trajectory `trajectory` rather than being calculated from `trajectory_kj`